Makefile

```diff
@@ -44,14 +42,12 @@ build-local:
 
 push:
 	@$(CONTAINER_ENGINE) push $(CONSOLE_IMAGE):latest
-	@$(CONTAINER_ENGINE) push $(CONSOLE_IMAGE):${VERSION}
 	@$(CONTAINER_ENGINE) push $(CONSOLE_IMAGE):${IMAGE_TAG}
 
-
-start-dev: export REACT_APP_CIQ_API_URL = http://localhost:8081
+start-dev: export VITE_CIQ_API_URL = http://localhost:8081
```

Comment:
Here we are defining the Backend URL var name as "VITE_CIQ_API_URL"
However, the var used on the `nginx.conf` file is called: "BACKEND_URL"

Reply:
I did this on purpose to separate the prod and dev environments + not expose the variable to the client

https://vite.dev/guide/env-and-mode#env-files